Orange/widgets/visualize/owmosaic.py

```diff
@@ -281,8 +281,11 @@ class OWMosaicDisplay(OWWidget):
     variable2 = ContextSetting("", exclude_metas=False)
     variable3 = ContextSetting("", exclude_metas=False)
     variable4 = ContextSetting("", exclude_metas=False)
+    variable_color = ContextSetting("", exclude_metas=False)
     selection = ContextSetting(set())
 
+    color_var = None
```

Comment:
This does work, but we define instance attributes in `__init__`. Settings are an exception.